include/extractor/extractor_config.hpp

```diff
@@ -71,7 +71,9 @@ struct ExtractorConfig
         rtree_nodes_output_path = basepath + ".osrm.ramIndex";
         rtree_leafs_output_path = basepath + ".osrm.fileIndex";
         edge_segment_lookup_path = basepath + ".osrm.edge_segment_lookup";
-        edge_penalty_path = basepath + ".osrm.edge_penalties";
+        turn_duration_penalties_path = basepath + ".osrm.turn_weight_penalties";
+        turn_weight_penalties_path = basepath + ".osrm.turn_duration_penalties";
```

Comment:
Looks like the values are swapped:
turn_weight_penalties → turn_duration_penalties and vice versa